website/content/docs/interoperability-matrix.mdx

```diff
@@ -0,0 +1,198 @@
+---
```

Comment:
This file doesn't exist anymore. We should make any updates to the relevant file under `/partners` (I'm assuming the `hsm.mdx` file.